torch/quantization/fx/quantization_patterns.py

```diff
@@ -597,9 +618,12 @@ def convert(self,
                     if isinstance(self.conv, torch.nn.intrinsic._FusedModule):
                         float_conv = self.conv[0]
                     assert qconfig is not None
-                    float_conv.weight_post_process = qconfig.weight()
+                    weight_post_process = qconfig.weight()
                     # run weight observer
-                    float_conv.weight_post_process(float_conv.weight)
+                    weight_post_process(float_conv.weight)
+                float_conv._weight_qparams = get_qparam_dict(weight_post_process)
```

Comment:
have we considered storing `scale` and `zero_point` as individual buffers, to avoid the need for custom serialization handling?

Reply:
hmm, looks like we can only store Optional[Tensor] in buffers: https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/module.py#L271, it works for `scale` and `zero_point`, but we also need to store qscheme and dtype, and I feel it might be a bit harder?